report.py

```diff
@@ -13,33 +13,11 @@ class MissingStudentException(Exception):
     pass
 
 
-class TeamReportListView(MethodView):
-    @login_required
-    def get(self, team_id):
-        """
-        Renders a list of available reports for a specific team.
-        PR #47 supersedes this but for now I'll leave the route in so we can test with it.
-
-        Keyword arguments:
-        team_id -- tid of the team to generate reports for
-        """
-        # Make list of students on this team
-        # This data gets passed straight to the template, which can handle having zero rows in the db result.
-        students = gbmodel.students.query.filter_by(tid=team_id, session_id=0)
-
-        sessions = {'first', 'second'}
-
-        teams = [(row.id, row.name) for row in gbmodel.teams.query.filter_by(session_id=0)]
-        # 404 if there's no such team
-        if len(teams) <= 0:
-            return make_response(render_template('404.html'), 404)
-
-        return render_template('reportList.html',
-                               students=students,
-                               sessions=sessions,
-                               teams=teams,
-                               team_id=team_id,
-                               session_id=0)
+class MissingTeamException(Exception):
```

Comment:
should be 'missingTeamException

Reply:
Are you sure? That's not how the existing class is written (nor how any of our classes are written?)